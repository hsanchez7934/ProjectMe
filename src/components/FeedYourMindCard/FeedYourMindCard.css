.info-card {
  border: 1px solid black;
  border-top-left-radius: 30px;
  border-bottom-right-radius: 30px;
  height: 550px;
  min-width: 430px;
  margin: 0px 40px 0px 0px;
  background-color: purple; }

.info-card-image-container {
  height: 65%; }

.info-card-image {
  height: 100%;
  width: 100%;
  border-top-left-radius: 30px; }

.info-card-title {
  text-align: center;
  background-color: magenta;
  margin: 0px 0px 0px 0px;
  padding: 20px 10px; }

.info-card-body-container {
  background-color: pink;
  display: none; }

.info-card-body {
  margin: 0; }
