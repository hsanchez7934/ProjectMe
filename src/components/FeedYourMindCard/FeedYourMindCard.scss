@import "../../styles/opensans.scss";
@import "../../styles/colors.scss";

.card {
  color: $paper;
  float: left;
  height: 550px;
  margin-bottom: 30px;
  min-width: 400px;
  padding: 0 1.7rem;
  .wrapper {
    border-bottom-right-radius: 30px;
    border-top-left-radius: 30px;
    box-shadow: 0 19px 38px rgba($black, 0.3), 0 15px 12px rgba($black, 0.2);
    min-height: 540px;
    overflow: hidden;
    position: relative;
    &:hover {
      .data {
        transform: translateY(-15px);
      }
    }
  }
  .data {
    bottom: -16px;
    position: absolute;
    transform: translateY(calc(70px + 1em));
    transition: transform 0.3s;
    width: 100%;
    .content {
      background-color: $black;
      box-shadow: 0 5px 30px 10px rgba($black, 0.3);
      padding: 1em;
      position: relative;
      opacity: 0.8;
      z-index: 1;
        .card-title {
          font-family: $open-sans;
          font-weight: 300;
          margin-top: 10px;
          overflow: scroll;
        }
        .card-description {
          font-weight: 300;
          height: 70px;
          margin: 0px 0px 0px 0px;
          overflow: scroll;
          padding-top: 15px;
        }
        a {
          color: $silk;
          display: block;
          font-family: $open-sans;
          padding-top: 10px;
          text-decoration: none;
        }
        a:hover {
          color: $palegold;
          font-weight: 600;
          text-decoration: underline;
        }
    }
  }
}
