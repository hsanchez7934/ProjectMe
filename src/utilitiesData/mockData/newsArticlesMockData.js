const newsArticlesMockData = [
  {
    author: `Sarah Gibbens`,
    title: `Endangered Siberian Tiger Returns From Exile`,
    description: `Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.`,
    url: `https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html`,
    urlToImage: `https://pmdvod.nationalgeographic.com/NG_Video/329/706/nw-dly-ds1702001-186-amur-tiger-release-vin_640x360_953828419671.jpg`,
    publishedAt: `2017-11-02T21:39:08Z`
  },

  {
    author: `Sarah Gibbens`,
    title: `Family of Travel Bloggers Found Alive After Pirate Attack`,
    description: `The Californian family of
                  four emerged from the Amazon
                  rainforest with only minor
                  bites and scratches.`,
    url: `https://www.nationalgeographic.com/adventure/destinations/south-america/family-american-travel-bloggers-found-brazil-pirates-spd.html`,
    urlToImage: `https://www.nationalgeographic.com/content/dam/adventure/photos/2017/stories/brazil-amazon-kidnapping/amazon-family-kidnapping-river-para.jpg`,
    publishedAt: `2017-11-02T17:29:25Z`
  }
];

export default newsArticlesMockData;
