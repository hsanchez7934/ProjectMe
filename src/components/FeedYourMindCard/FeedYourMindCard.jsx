import React from 'react';
import PropTypes from 'prop-types';
import './FeedYourMindCard.css';

const backgroundStyle = (imageUrl) => ({
  backgroundImage: 'url(' + imageUrl + ')',
  backgroundSize: 'cover',
  backgroundPosition: 'center',
  backgroundRepeat: 'no-repeat'
});

const FeedYourMindCard = ({ article }) => {
  const { title, description, url, urlToImage } = article;
  return (
    <article className="card">
      <div
        className="wrapper"
        style={backgroundStyle(urlToImage)}>
        <div className="data">
          <div className="content">
            <h3 className="card-title">
              {title}
            </h3>
            <p className="card-description">
              {description}
            </p>
            <a href={url}>Click here for full article</a>
          </div>
        </div>
      </div>
    </article>
  );
};

FeedYourMindCard.propTypes = {
  article: PropTypes.object
};

export default FeedYourMindCard;
