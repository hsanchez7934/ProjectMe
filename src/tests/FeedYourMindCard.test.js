import React from 'react';
import { mount } from 'enzyme';
import config from './testSetup.js';
import FeedYourMindCard from '../components/FeedYourMindCard/FeedYourMindCard.jsx';
import newsArticlesMockData from '../utilitiesData/mockData/newsArticlesMockData.js';

describe(`FeedYourMindCard units testing`, () => {
  let wrapper;
  let article;
  let div1;
  let div2;
  let div3;
  let h3;
  let pNode;
  let aNode;

  beforeEach(() => {
    wrapper = mount(
      <FeedYourMindCard
        article={newsArticlesMockData[0]} />
    );

    article = wrapper.find('article');
    div1 = article.find('div').at(0);
    div2 = article.find('div').at(1);
    div3 = article.find('div').at(2);
    h3 = article.find('h3');
    pNode = article.find('p');
    aNode = article.find('a');

  });

  test(`should create an instance of
        FeedYourMindCard`, () => {
      expect(wrapper.exists()).toEqual(true);
    });

  test(`should render one paret article node, 3 three divs,
        one h3, one p node, and one a node`, () => {
      expect(article.exists()).toEqual(true);
      expect(div1.exists()).toEqual(true);
      expect(div2.exists()).toEqual(true);
      expect(div3.exists()).toEqual(true);
      expect(h3.exists()).toEqual(true);
      expect(pNode.exists()).toEqual(true);
      expect(aNode.exists()).toEqual(true);
    });

  test(`should match snapshot`, () => {
    expect(wrapper).toMatchSnapshot();
  });
});
