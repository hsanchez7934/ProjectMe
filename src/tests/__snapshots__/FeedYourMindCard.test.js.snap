// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeedYourMindCard units testing should match snapshot 1`] = `
ReactWrapper {
  "length": 1,
  Symbol(enzyme.__unrendered__): <FeedYourMindCard
    article={
      Object {
        "author": "Sarah Gibbens",
        "description": "Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.",
        "publishedAt": "2017-11-02T21:39:08Z",
        "title": "Endangered Siberian Tiger Returns From Exile",
        "url": "https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html",
        "urlToImage": "https://pmdvod.nationalgeographic.com/NG_Video/329/706/nw-dly-ds1702001-186-amur-tiger-release-vin_640x360_953828419671.jpg",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "function",
    "props": Object {
      "article": Object {
        "author": "Sarah Gibbens",
        "description": "Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.",
        "publishedAt": "2017-11-02T21:39:08Z",
        "title": "Endangered Siberian Tiger Returns From Exile",
        "url": "https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html",
        "urlToImage": "https://pmdvod.nationalgeographic.com/NG_Video/329/706/nw-dly-ds1702001-186-amur-tiger-release-vin_640x360_953828419671.jpg",
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": <article
        class="card"
      >
        <div
          class="wrapper"
          style="background-image: url(https://pmdvod.nationalgeographic.com/NG_Video/329/706/nw-dly-ds1702001-186-amur-tiger-release-vin_640x360_953828419671.jpg); background-size: cover; background-position: center; background-repeat: no-repeat;"
        >
          <div
            class="data"
          >
            <div
              class="content"
            >
              <h3
                class="card-title"
              >
                Endangered Siberian Tiger Returns From Exile
              </h3>
              <p
                class="card-description"
              >
                Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.
              </p>
              <a
                href="https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html"
              >
                Click here for full article
              </a>
            </div>
          </div>
        </div>
      </article>,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="wrapper"
          style={
            Object {
              "backgroundImage": "url(https://pmdvod.nationalgeographic.com/NG_Video/329/706/nw-dly-ds1702001-186-amur-tiger-release-vin_640x360_953828419671.jpg)",
              "backgroundPosition": "center",
              "backgroundRepeat": "no-repeat",
              "backgroundSize": "cover",
            }
          }
        >
          <div
            className="data"
          >
            <div
              className="content"
            >
              <h3
                className="card-title"
              >
                Endangered Siberian Tiger Returns From Exile
              </h3>
              <p
                className="card-description"
              >
                Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.
              </p>
              <a
                href="https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html"
              >
                Click here for full article
              </a>
            </div>
          </div>
        </div>,
        "className": "card",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": <div
            class="wrapper"
            style="background-image: url(https://pmdvod.nationalgeographic.com/NG_Video/329/706/nw-dly-ds1702001-186-amur-tiger-release-vin_640x360_953828419671.jpg); background-size: cover; background-position: center; background-repeat: no-repeat;"
          >
            <div
              class="data"
            >
              <div
                class="content"
              >
                <h3
                  class="card-title"
                >
                  Endangered Siberian Tiger Returns From Exile
                </h3>
                <p
                  class="card-description"
                >
                  Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.
                </p>
                <a
                  href="https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html"
                >
                  Click here for full article
                </a>
              </div>
            </div>
          </div>,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="data"
            >
              <div
                className="content"
              >
                <h3
                  className="card-title"
                >
                  Endangered Siberian Tiger Returns From Exile
                </h3>
                <p
                  className="card-description"
                >
                  Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.
                </p>
                <a
                  href="https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html"
                >
                  Click here for full article
                </a>
              </div>
            </div>,
            "className": "wrapper",
            "style": Object {
              "backgroundImage": "url(https://pmdvod.nationalgeographic.com/NG_Video/329/706/nw-dly-ds1702001-186-amur-tiger-release-vin_640x360_953828419671.jpg)",
              "backgroundPosition": "center",
              "backgroundRepeat": "no-repeat",
              "backgroundSize": "cover",
            },
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": <div
                class="data"
              >
                <div
                  class="content"
                >
                  <h3
                    class="card-title"
                  >
                    Endangered Siberian Tiger Returns From Exile
                  </h3>
                  <p
                    class="card-description"
                  >
                    Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.
                  </p>
                  <a
                    href="https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html"
                  >
                    Click here for full article
                  </a>
                </div>
              </div>,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="content"
                >
                  <h3
                    className="card-title"
                  >
                    Endangered Siberian Tiger Returns From Exile
                  </h3>
                  <p
                    className="card-description"
                  >
                    Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.
                  </p>
                  <a
                    href="https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html"
                  >
                    Click here for full article
                  </a>
                </div>,
                "className": "data",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": <div
                    class="content"
                  >
                    <h3
                      class="card-title"
                    >
                      Endangered Siberian Tiger Returns From Exile
                    </h3>
                    <p
                      class="card-description"
                    >
                      Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.
                    </p>
                    <a
                      href="https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html"
                    >
                      Click here for full article
                    </a>
                  </div>,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <h3
                        className="card-title"
                      >
                        Endangered Siberian Tiger Returns From Exile
                      </h3>,
                      <p
                        className="card-description"
                      >
                        Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.
                      </p>,
                      <a
                        href="https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html"
                      >
                        Click here for full article
                      </a>,
                    ],
                    "className": "content",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": <h3
                        class="card-title"
                      >
                        Endangered Siberian Tiger Returns From Exile
                      </h3>,
                      "key": null,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Endangered Siberian Tiger Returns From Exile",
                        "className": "card-title",
                      },
                      "ref": null,
                      "rendered": Array [
                        "Endangered Siberian Tiger Returns From Exile",
                      ],
                      "type": "h3",
                    },
                    Object {
                      "instance": <p
                        class="card-description"
                      >
                        Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.
                      </p>,
                      "key": null,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.",
                        "className": "card-description",
                      },
                      "ref": null,
                      "rendered": Array [
                        "Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.",
                      ],
                      "type": "p",
                    },
                    Object {
                      "instance": <a
                        href="https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html"
                      >
                        Click here for full article
                      </a>,
                      "key": null,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Click here for full article",
                        "href": "https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html",
                      },
                      "ref": null,
                      "rendered": Array [
                        "Click here for full article",
                      ],
                      "type": "a",
                    },
                  ],
                  "type": "div",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "article",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "function",
      "props": Object {
        "article": Object {
          "author": "Sarah Gibbens",
          "description": "Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.",
          "publishedAt": "2017-11-02T21:39:08Z",
          "title": "Endangered Siberian Tiger Returns From Exile",
          "url": "https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html",
          "urlToImage": "https://pmdvod.nationalgeographic.com/NG_Video/329/706/nw-dly-ds1702001-186-amur-tiger-release-vin_640x360_953828419671.jpg",
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": <article
          class="card"
        >
          <div
            class="wrapper"
            style="background-image: url(https://pmdvod.nationalgeographic.com/NG_Video/329/706/nw-dly-ds1702001-186-amur-tiger-release-vin_640x360_953828419671.jpg); background-size: cover; background-position: center; background-repeat: no-repeat;"
          >
            <div
              class="data"
            >
              <div
                class="content"
              >
                <h3
                  class="card-title"
                >
                  Endangered Siberian Tiger Returns From Exile
                </h3>
                <p
                  class="card-description"
                >
                  Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.
                </p>
                <a
                  href="https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html"
                >
                  Click here for full article
                </a>
              </div>
            </div>
          </div>
        </article>,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="wrapper"
            style={
              Object {
                "backgroundImage": "url(https://pmdvod.nationalgeographic.com/NG_Video/329/706/nw-dly-ds1702001-186-amur-tiger-release-vin_640x360_953828419671.jpg)",
                "backgroundPosition": "center",
                "backgroundRepeat": "no-repeat",
                "backgroundSize": "cover",
              }
            }
          >
            <div
              className="data"
            >
              <div
                className="content"
              >
                <h3
                  className="card-title"
                >
                  Endangered Siberian Tiger Returns From Exile
                </h3>
                <p
                  className="card-description"
                >
                  Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.
                </p>
                <a
                  href="https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html"
                >
                  Click here for full article
                </a>
              </div>
            </div>
          </div>,
          "className": "card",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": <div
              class="wrapper"
              style="background-image: url(https://pmdvod.nationalgeographic.com/NG_Video/329/706/nw-dly-ds1702001-186-amur-tiger-release-vin_640x360_953828419671.jpg); background-size: cover; background-position: center; background-repeat: no-repeat;"
            >
              <div
                class="data"
              >
                <div
                  class="content"
                >
                  <h3
                    class="card-title"
                  >
                    Endangered Siberian Tiger Returns From Exile
                  </h3>
                  <p
                    class="card-description"
                  >
                    Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.
                  </p>
                  <a
                    href="https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html"
                  >
                    Click here for full article
                  </a>
                </div>
              </div>
            </div>,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="data"
              >
                <div
                  className="content"
                >
                  <h3
                    className="card-title"
                  >
                    Endangered Siberian Tiger Returns From Exile
                  </h3>
                  <p
                    className="card-description"
                  >
                    Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.
                  </p>
                  <a
                    href="https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html"
                  >
                    Click here for full article
                  </a>
                </div>
              </div>,
              "className": "wrapper",
              "style": Object {
                "backgroundImage": "url(https://pmdvod.nationalgeographic.com/NG_Video/329/706/nw-dly-ds1702001-186-amur-tiger-release-vin_640x360_953828419671.jpg)",
                "backgroundPosition": "center",
                "backgroundRepeat": "no-repeat",
                "backgroundSize": "cover",
              },
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": <div
                  class="data"
                >
                  <div
                    class="content"
                  >
                    <h3
                      class="card-title"
                    >
                      Endangered Siberian Tiger Returns From Exile
                    </h3>
                    <p
                      class="card-description"
                    >
                      Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.
                    </p>
                    <a
                      href="https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html"
                    >
                      Click here for full article
                    </a>
                  </div>
                </div>,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": <div
                    className="content"
                  >
                    <h3
                      className="card-title"
                    >
                      Endangered Siberian Tiger Returns From Exile
                    </h3>
                    <p
                      className="card-description"
                    >
                      Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.
                    </p>
                    <a
                      href="https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html"
                    >
                      Click here for full article
                    </a>
                  </div>,
                  "className": "data",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": <div
                      class="content"
                    >
                      <h3
                        class="card-title"
                      >
                        Endangered Siberian Tiger Returns From Exile
                      </h3>
                      <p
                        class="card-description"
                      >
                        Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.
                      </p>
                      <a
                        href="https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html"
                      >
                        Click here for full article
                      </a>
                    </div>,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        <h3
                          className="card-title"
                        >
                          Endangered Siberian Tiger Returns From Exile
                        </h3>,
                        <p
                          className="card-description"
                        >
                          Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.
                        </p>,
                        <a
                          href="https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html"
                        >
                          Click here for full article
                        </a>,
                      ],
                      "className": "content",
                    },
                    "ref": null,
                    "rendered": Array [
                      Object {
                        "instance": <h3
                          class="card-title"
                        >
                          Endangered Siberian Tiger Returns From Exile
                        </h3>,
                        "key": null,
                        "nodeType": "host",
                        "props": Object {
                          "children": "Endangered Siberian Tiger Returns From Exile",
                          "className": "card-title",
                        },
                        "ref": null,
                        "rendered": Array [
                          "Endangered Siberian Tiger Returns From Exile",
                        ],
                        "type": "h3",
                      },
                      Object {
                        "instance": <p
                          class="card-description"
                        >
                          Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.
                        </p>,
                        "key": null,
                        "nodeType": "host",
                        "props": Object {
                          "children": "Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.",
                          "className": "card-description",
                        },
                        "ref": null,
                        "rendered": Array [
                          "Vladik was once forcibly
                  removed from the Russian city
                  Vladivostok. Now, one year,
                  400 miles, and three dead bears
                  later, he's back.",
                        ],
                        "type": "p",
                      },
                      Object {
                        "instance": <a
                          href="https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html"
                        >
                          Click here for full article
                        </a>,
                        "key": null,
                        "nodeType": "host",
                        "props": Object {
                          "children": "Click here for full article",
                          "href": "https://news.nationalgeographic.com/2017/11/vladik-vladivostok-russia-far-east-siberian-tiger-spd.html",
                        },
                        "ref": null,
                        "rendered": Array [
                          "Click here for full article",
                        ],
                        "type": "a",
                      },
                    ],
                    "type": "div",
                  },
                ],
                "type": "div",
              },
            ],
            "type": "div",
          },
        ],
        "type": "article",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
